tests/rptest/tests/partition_reassignments_test.py

```diff
@@ -264,6 +265,42 @@ def initial_setup_steps(self,
 
         return initial_assignments, all_node_idx, producers
 
+    def execute_reassign_partitions(self,
+                                    reassignments: dict,
+                                    timeout_s: int = 10):
+        kafka_tools = KafkaCliTools(self.redpanda)
+        return kafka_tools.reassign_partitions(
+            reassignments=reassignments,
+            operation="execute",
+            # RP may report a problem with the number of replicas, this can
+            # happen when the recieving broker has out-of-date metadata. So
+            # retry the request a few times.
```

Comment:
This comment is out of date

Reply:
I don't believe it is out-of-date. It is, however, incorrect. The comment should instead refer to unknown_topic_or_partition instead of replication factor.